### ISSUE TITLE
SASS strips brackets from grid-template expression

### ISSUE THREAD

**Issue by gajus** [sass/node-sass#1967]
```bash
% npm -v
4.5.0
% node -v
v7.9.0
% node -p process.versions
{ http_parser: '2.7.0',
  node: '7.9.0',
  v8: '5.5.372.43',
  uv: '1.11.0',
  zlib: '1.2.11',
  ares: '1.10.1-DEV',
  modules: '51',
  openssl: '1.0.2k',
  icu: '58.2',
  unicode: '9.0',
  cldr: '30.0.3',
  tz: '2016j' }
% node -p process.platform
darwin
% node -p process.arch
x64
% node -p "require('node-sass').info
>
% node -p "require('node-sass').info"
node-sass	4.5.2	(Wrapper)	[JavaScript]
libsass  	3.5.0.beta.2	(Sass Compiler)	[C/C++]
% npm ls node-sass
/Users/gajus/Documents/dev/applaudience/applaudience-com
└── node-sass@4.5.2

```

Input:

```css
.test {
  grid-template:
    [header-left] "head head" 30px [header-right]
    [main-left]   "nav  main" 1fr  [main-right]
    [footer-left] "nav  foot" 30px [footer-right]
    / 120px 1fr;
}

```

Output:

```css
.test {
  grid-template: [header-left] "head head" 30px [header-right] [main-left] "nav  main" 1fr [main-right] [footer-left] "nav  foot" 30px footer-right/120px 1fr; }
```

(Notice the missing brackets around "footer-right".)

Expected output:

```css
.test {
  grid-template: [header-left] "head head" 30px [header-right] [main-left] "nav  main" 1fr [main-right] [footer-left] "nav  foot" 30px "footer-right"/120px 1fr; }

```

**Comment by nschonni** [comment]
Can you file an issue with https://github.com/sass/libsass/issues and link it back here, thanks!

**Comment by xzyfer** [comment]
I have confirmed this is an issue with LibSass

**Comment by xzyfer** [comment]
Fixed in 4.8.0
